db/migrate/20180426030117_add_role_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :role, :string, null: false, default: 'member'
+  end
+end
```

Comment:
You can combine this `add_column` with the other migration you made, such that you have two add_columns in one migration